packages/text/test/TextMetrics.js

```diff
@@ -485,6 +485,31 @@ describe('PIXI.TextMetrics', function ()
 
             expect(bool).to.equal(false);
         });
+
+        it('overridable breaking spaces', function ()
+        {
+            const reg = /[あいうえお]/;
+
+            // override breakingSpace
+            TextMetrics.isBreakingSpace = function (char, nextChar)
```

Comment:
After this test is over, can you reset isBreakingSpace to the default. Otherwise it may have unintended side effects for other tests.